api/src/models/website.model.js

```diff
@@ -37,6 +37,16 @@ const websiteSchema = new mongoose.Schema(
         enum: sortingOptions,
       },
     },
+    colour: {
+      type: Number,
+      required: true,
+      default: 1,
+    },
+    layout: {
+      type: Number,
+      required: true,
+      default: 1,
+    },
```

Comment:
If these are marked as `required` then it might be a good idea to implement the controller functionality in this PR as well (which should be simply updating these two fields based). Also since these are essentially enum constants we should follow the convention we've been using for enum fields in other models.

@pmyip3 can be a helpful source of information here.

https://github.com/Researchify/Researchify/blob/main/api/src/models/website.model.js#L32